src/mint.rs

```diff
@@ -131,8 +131,7 @@ impl ReissueTransaction {
 #[derive(Eq, PartialEq, Debug, Clone, Deserialize, Serialize)]
 pub struct ReissueRequest {
     pub transaction: ReissueTransaction,
-    // Signatures from the owners of each input, signing `self.transaction.blinded().hash()`
-    pub input_ownership_proofs: HashMap<SpendKey, blsttc::Signature>,
+    pub spent_proofs: BTreeMap<SpendKey, SpentProof>,
```

Comment:
We are mixing present and past tense here:   SpendKey and SpentProof.

Perhaps SpendKey should be renamed to SpentKey?     It is a little bit weird though since a SpentKey is first generated by client before dbc is actually written to spentbook (spent).

I'm not sure what the right answer is.  just raising for consideration.

Reply:
yep, not great.

How about `DBC SpendingKey` in-place of `DBC SpendKey` and Spent for everything else: `SpentSig`/`SpentBook`/`SpentProof`?

Reply:
That could be...  Seems about the same meaning as SpendKey though.

is it terrible to just call it SpentKey, so we are past tense with everything?

In a sense it *is* a spent key, as its basically meaningless unless/until actually stored in the SpentBook.

but I'm ok whichever way you decide to go.

Reply:
I'd rather go the other way, make them all use the Spend* prefix

    SpendKey, SpendBook, SpendProof, SpendSig

but.. I don't want to make this change in this PR. Lets keep it as is for now

Reply:
> make them all use the Spend* prefix

yeah, I'd be good with that.